google-cloud-spanner-hibernate-samples/microprofile-jpa-sample/pom.xml

```diff
@@ -57,6 +57,12 @@
       <version>4.1.0</version>
     </dependency>
 
+    <dependency>
+        <groupId>io.undertow</groupId>
+        <artifactId>undertow-core</artifactId>
+        <version>2.3.11.Final</version>
```

Comment:
Why do we need to add this dependnecy, is this because Maven can pull the latest version of `undertow-core`, rather than the version imported from `thorntail`?